internal/compiler/generator/rust.rs

```diff
@@ -1384,13 +1408,11 @@ fn generate_item_tree(
         impl #inner_component_id {
             pub fn new(#(parent: #parent_component_type)*) -> #new_result {
                 #![allow(unused)]
-                #create_window_adapter // We must create the window first to initialize the backend before using the style
```

Comment:
This comment is suspicion.  We then won't initialize the backend before using the style anymore?  This may cause problems with the NativeStyleMetrics and such

Reply:
Ah, this originates from https://github.com/slint-ui/slint/commit/f0b047057b7d73288a0d518a8c499652ae4a3188 - I need to try that. Perhaps we can still initialise the backend here anyway.